test_istgt.sh

```diff
@@ -518,25 +532,569 @@ run_lu_rf_test ()
 
 	sleep 5
 
-	kill -9 $replica3_pid
-	start_replica -i "$CONTROLLER_IP" -p "$CONTROLLER_PORT" -I "$replica3_ip" -P "$(($replica3_port + 10))" -V $replica3_vdev &
+	pkill -9 -P $replica3_pid
+	start_replica -i "$CONTROLLER_IP" -p "$CONTROLLER_PORT" -I "$replica3_ip" -P "$(($replica3_port + 10))" -V $replica3_vdev -q "$QUORUM" &
 	replica3_pid=$!
-	sleep 5
+        sleep 5
 
-	wait $replica3_pid
-	if [ $? -eq 0 ]; then
+	if ps -p $replica3_pid > /dev/null 2>&1
```

Comment:
what is this?

Reply:
It checks whether the process exists or not.